test/unit/elasticsearch_index_test.rb

```diff
@@ -282,6 +282,35 @@ def test_should_populate_tags_field
     assert_requested(:post, "http://example.com:9200/test-index/_bulk")
   end
 
+  def test_should_populate_mainstream_browse_pages_field
+    stub_popularity_index_requests(["/foo/bar"], 1.0)
+
+    json_document = {
+      "_type" => "edition",
+      "link" => "/foo/bar",
+      "section" => "benefits",
+      "subsection" => "entitlement",
+    }
+    document = stub("document", elasticsearch_export: json_document)
+
+    # Note that this comes with a trailing newline, which elasticsearch needs
+    payload = <<-eos
+{"index":{"_type":"edition","_id":"/foo/bar"}}
+{"_type":"edition","link":"/foo/bar","section":"benefits","subsection":"entitlement","popularity":1.0,"mainstream_browse_pages":["benefits/entitlement"],"tags":[]}
+    eos
+    response = <<-eos
+{"took":5,"items":[
+  { "index": { "_index":"test-index", "_type":"edition", "_id":"/foo/bar", "ok":true } }
+]}
+    eos
+    stub_request(:post, "http://example.com:9200/test-index/_bulk").with(
+        body: payload,
+        headers: {"Content-Type" => "application/json"}
+    ).to_return(body: response)
+    @wrapper.add [document]
+    assert_requested(:post, "http://example.com:9200/test-index/_bulk")
```

Comment:
I believe `stub_request` returns an object you can use for assertions.  I think this would work:
```
bulk_request = stub_request(:post, "http://example.com:9200/test-index/_bulk").with(
    body: payload,
    headers: {"Content-Type" => "application/json"}
).to_return(body: response)

@wrapper.add([document])

assert_requested(bulk_request)
```